source/enumerate-collections.rst

```diff
@@ -271,6 +271,11 @@ MongoDB 4.0 introduced a ``nameOnly`` boolean option to the ``listCollections``
 database command, which limits the command result to only include collection
 names. NOTE: ``nameOnly`` is applied before any filter is applied.
 
+MongoDB 4.0 also added an ``authorizedCollections`` boolean option to the ``listCollections``
+command, which can be used to limit the command result to only include collections
+the user is authorized to use. Drivers MAY allow users to set the ``authorizedCollections``
+option on the ``listCollectionNames`` method.
```

Comment:
IMO, this paragraph would read better if placed at the end of this section. The list below and subsequent paragraph are still talking about `nameOnly`.

Reply:
done